local-cluster/tests/local_cluster_slow_1.rs

```diff
@@ -386,6 +391,191 @@ fn test_kill_partition_switch_threshold_progress() {
     );
 }
 
+#[test]
+#[serial]
+#[allow(unused_attributes)]
+fn test_oc_bad_signatures() {
```

Comment:
any reason to put this in `local_cluster_slow` straight away? we only need 10 votes - does it take that long?

Reply:
yeah you're right, will move it over